### PR TITLE
Bugfix min_ess

### PR DESCRIPTION
**Pull request by HarrisonS-Phys**
Bug fix for min_ess runs, the recursion of fit.run() currently seems to be broken following the latest changes that were made to run() and models to allow for new sampling of priors

### CODE REVIEW

**Comment by maxisi** [comment]
@HarrisonS-Phys please resolve conflicts with main branch before this can be merged

**Comment by HarrisonS-Phys** [comment]
Should be fixed

**Comment by HarrisonS-Phys** [comment]
@maxisi  you mentioned in person that there were some changes you'd like to see before merging, are there comments for those changes available on github?

**Comment by maxisi** [comment]
@HarrisonS-Phys yes, do you see those right above your last comment?

**Review by maxisi — CHANGES_REQUESTED** [review]
@HarrisonS-Phys please see these review comments

File: ringdown/fit.py [hunk 1]
```diff
@@ -690,11 +690,13 @@ def run(self, prior=False, suppress_warnings=True, store_residuals=True,
 
         min_ess: number
             if given, keep re-running the sampling with longer chains until the
-            minimum effective sample size exceeds `min_ess` (def. `None`).
+            minimum effective sample size exceeds `min_ess` (def. `-0.1`).
```
[the remark this answers is not in the document]
Comment (HarrisonS-Phys) [reply ringdown/fit.py]: I kept the default as None, and now set min_ess to be zero if None is passed.

**Comment by HarrisonS-Phys** [comment]
I think all the changes from your review have been applied.

**Pull request merged**